sats-dna/build.gradle.kts

```diff
@@ -57,17 +54,6 @@ android {
         targetCompatibility = JavaVersion.VERSION_11
     }
 
-    composeOptions {
-        kotlinCompilerExtensionVersion = libs.versions.compose.compiler.get()
-    }
-
-    kotlinOptions {
-        freeCompilerArgs += listOf(
-            "-P",
-            "plugin:androidx.compose.compiler.plugins.kotlin:strongSkipping=true",
```

Comment:
Since this was removed, should there be a `composeCompiler` block with `enableStrongSkippingMode = true`?